### DISCUSSION TITLE
Sparse checkout

### DISCUSSION THREAD

**Discussion by raisen** [nektos/act#1923] [Q&A]
When using sparse-checkout from actions/checkout@v3, it seems act is copying the entire folder as I see the docker cp command output.
Is sparse checkout compatible with act?

**Comment by ChristopherHX** [comment]
Use `--no-skip-checkout` than sparse checkout will work, but it will download from GitHub.

_I'm not shure if `github.sha` and `github.repository` are correct in your case in act, if not you have to add help act by hardcoding or `repository: ${{ env.ACT && 'owner/repo' || github.repository }}` and `ref: ${{ env.ACT && 'ref/to/clone' || github.sha }}`_
